src/main/resources/db/changelog/changes/19-adding-foreign-keys-to-order-item-table.yaml

```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
+  - changeSet:
+      id: add_foreign_keys_to_order_item_table
+      author: mhorianin
+      changes:
+        - addForeignKeyConstraint:
+            baseTableName: order_item
+            baseColumnNames: order_id
+            referencedTableName: orders
+            referencedColumnNames: id
+            constraintName: fk_order_order_item
+            onDelete: CASCADE
```

Comment:
```suggestion
            onDelete: CASCADE
            
```

Reply:
I did not understand this comment. Should I delete this line?

Reply:
add empty line